client/notifications/notification.js

```diff
@@ -36,6 +36,34 @@ Meteor.startup(function() {
 					KonchatNotification.showDesktop(notification);
 				}
 			});
+
+			RocketChat.Notifications.onUser('audioNotification', function(notification) {
+
+				let openedRoomId = undefined;
+				if (['channel', 'group', 'direct'].includes(FlowRouter.getRouteName())) {
+					openedRoomId = Session.get('openedRoom');
+				}
```

Comment:
we can always get the openedRoomId from session and compare below comparing with `!= null` too